/**
 * We need change confirm_form to organic group by client petition.
 *
 * See more https://redmine.emergya.es/issues/115406#change-455714
 */
diff --git a/sites/all/modules/patched/og/og_ui/og_ui.pages.inc b/sites/all/modules/patched/og/og_ui/og_ui.pages.inc
index 51a05a9..e5a7c97 100644
--- a/sites/all/modules/patched/og/og_ui/og_ui.pages.inc
+++ b/sites/all/modules/patched/og/og_ui/og_ui.pages.inc
@@ -236,7 +236,7 @@ function og_ui_confirm_unsubscribe($form, &$form_state, $group_type, $group) {
 
   $label = entity_label($group_type, $group);
   $url = entity_uri($group_type, $group);
-  return confirm_form($form, t('Are you sure you want to unsubscribe from the group %title?', array('%title' => $label)), $url, NULL, t('Remove'), t('Cancel'));
+  return confirm_form($form, t('Are you sure you want to unsubscribe from the group %title?', array('%title' => $label)), $url, NULL, t('Accept'), t('Cancel'));
 }
 
 /**
